.github/workflows/run_ec2_pcluster.yaml

```diff
@@ -0,0 +1,195 @@
+
+name: test-ufs-bundle-pcluster
+
+# Controls when the workflow will run
+on:
+  schedule:
+    - cron: '0 4 * * *'
+  pull_request:
+    # pull request to develop
+    branches: [develop]
+  workflow_dispatch:
+    inputs:
+      ufs-bundle-branch:
+        description: 'The ufs-bundle branch to use. Default is "develop"'
+        required: true
+        default: 'develop'
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  test-ufs-bundle:
+    runs-on: [self-hosted, Linux, X64, pcluster-ci-20230717]
+
+    steps:
+      - name: cleanup
+        if: ${{ github.event_name == 'schedule' }}
```

Comment:
Only start from scratch for nightly runs, not for PR / workflow dispatch runs during the day